profiles/common/modules/features/nexteuropa_feature_set/nexteuropa_feature_set.module

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * @file
+ * Implements some specific logic for platform feature sets.
+ */
+
+/**
+ * Implements hook_FORM_ID_alter().
+ */
+function nexteuropa_feature_set_form_feature_set_admin_form_alter(&$form, &$form_state) {
```

Comment:
Should we provide this permission implementation in the scope of the NEPT-182?
This looks more like a task for a separate ticket. Correct me if I'm wrong.